### PR TITLE
Initialize variable p (#287)

### PR DESCRIPTION
**Pull request by jijoongmoon**
Initialize variable P and change error messages depends on it.

### CODE REVIEW

**Comment by jijoongmoon** [comment]
I think it is ready for review. Thanks!

File: tests/static_checking/static_check_bounds_cast.c [hunk 1]
```diff
@@ -42,21 +40,21 @@ extern void f2() {
 }
 
 extern void f3() {
-  int *p;
+  int p[10];
```
Comment (dtarditi) [inline tests/static_checking/static_check_bounds_cast.c]: This now gives errors about assigning to p.   It is illegal in C to assign to variables with array type.  Could you leave p as a pointer variable and initialize it instead:
```
int *p = 0;
```

File: tests/static_checking/static_check_bounds_cast.c [hunk 2]
```diff
@@ -100,28 +98,28 @@ extern int *f7(int arr checked[]) {
 
 extern void f18(int i) {
   int c;
-  int *p;
+  int p[10];
```
Comment (dtarditi) [inline tests/static_checking/static_check_bounds_cast.c]: This has the same issues as the prior comment.

File: tests/static_checking/static_check_bounds_cast.c [hunk 3]
```diff
@@ -10,18 +10,16 @@ extern void f1() {
   int i;
   ptr<int> c = 0;
   int b[10];
-
-  int *p;
+  int p[10];
```
Comment (dtarditi) [inline tests/static_checking/static_check_bounds_cast.c]: This change is looks good.

**Comment by dtarditi** [comment]
This pull request has not been updated.  I am closing it and will do a separate pull request.

**Pull request closed without merging**